src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -23,6 +23,16 @@ namespace Microsoft.IdentityModel.JsonWebTokens
     /// </summary>
     public class JsonWebTokenHandler : TokenHandler
     {
+        /// <summary>
+        /// Default claim type mapping for inbound claims.
+        /// </summary>
+        public readonly static IDictionary<string, string> DefaultInboundClaimTypeMap = ClaimTypeMapping.InboundClaimTypeMap;
```

Comment:
This sets DefaultInboundClaimTypeMap to a reference of ClaimTypeMapping.InboundClaimTypeMap.
Should change the code here and in JwtSecurityTokenHandler. With only JwtSecurityTokenHandler there was no problem.
```cs
public readonly static IDictionary<string, string> DefaultInboundClaimTypeMap = new Dictionary<string,string>(ClaimTypeMapping.InboundClaimTypeMap);
```

The following code will result in JsonWebTokenHandler.DefaultInboundClaimTypeMap with size 0.
```cs
var jwtClaimsMapping = JwtSecurityTokenHandler.DefaultInboundClaimTypeMap;
jwtClaimsMapping.Clear();
var jsonClaimsMapping = JsonWebTokenHandler.DefaultInboundClaimTypeMap;
```

Reply:
Make sure there are tests to ensure these are independent to guard against regressions.

Reply:
Both Handlers use `new` to construct their Dictionaries, and a test checks for independence.